upgrading_appliances/index.md

```diff
@@ -2,13 +2,14 @@
 ---
 
 # Upgrading Appliances
+
 In this example, we will update from jansa to kasparov.
 
 1. Ensure that the appliance is up to date on the current branch
 
         # dnf update
 
-2. Install the new manageiq-release RPM for the version that you want to upgrade to. Refer to ([RPM URLs for manageiq-release](#rpm-urls-for-manageiq-release)) for the list of RPM URLs
+2. Install the new manageiq-release RPM for the version that you want to upgrade to. Refer to the [release directory](https://rpm.manageiq.org/index.html?prefix=release/) in our RPM repository to find the latest manageiq-release rpm. The number is the alphabetical number of the release (e.g. j (jansa) is the 10th letter: 10-jansa, k (kasparov) is the 11th letter: 11-kasparov, etc.)
```

Comment:
Would it make more sense to generate some other URL that links to the release rpm directly?  Maybe something like `https://rpm.manageiq.org/15.rpm`?

Reply:
like a redirect URL?  or actual RPM location.

Reply:
I was thinking it would be a copy of the actual RPM

Reply:
I like the idea, but let's discuss in a separate issue.  For now, I want to just remove the existing URLs.

Reply:
Created https://github.com/ManageIQ/manageiq-rpm_build/issues/367 to discuss there.